util/java/tmxviewer-java/README.MD

```diff
@@ -4,12 +4,12 @@ Java TMX Viewer example
 This is a simple example application showing how to use libtiled-java to load
 and view a Tiled map file.
 
-Trying it out in IntelliJ IDEA
+Opening the project in a modern IDE
 ------------------------------
```

Comment:
Please adjust the line to match the length of the title (or change the file to use `##` and `###`).